rules/rules.js

```diff
@@ -108,6 +108,55 @@ const removeRule = function (uid) {
     }
 };
 
+/**
+ * Runs the rule with the given UID. Throws errors when the rule doesn't exist
+ * or is unable to run (e.g. it's disabled);
+ * 
+ * @memberOf rules
+ * @param {String} uid the UID of the rule to run
+ * @param {Map<Object>} args optional dict of data to pass to the called rule
+ * @param {Boolean} cond when true, the called rull will only run if it's conditions are met
+ */
+const runRule = function (uid, args = {}, cond = true) {
+    const status = RuleManager.getStatus(uid);
+    if(!status) {
+        throw 'There is no rule with UID ' + uid;
```

Comment:
May we use `throw Error()` instead of `throw ()` for consistency across openhab-js?

Reply:
Done